packages/legend-query/src/components/QueryBuilderFilterPanel.tsx

```diff
@@ -188,9 +193,42 @@ const QueryBuilderFilterConditionEditor = observer(
     // Drag and Drop on filter condition value
     const handleDrop = useCallback(
       (item: QueryBuilderParameterDragSource): void => {
-        node.condition.setValue(item.variable.parameter);
+        const NUMERIC_PRIMITIVE_TYPES = [
```

Comment:
Seems like we repeat this piece of logic a lot in the operators. Let's try to see if we could abstract this logic out. Thanks!